src/fswAlgorithms/attControl/mrpProportionalDerivative/_UnitTest/Support/results_MRP_PD.py

```diff
@@ -0,0 +1,46 @@
+#
```

Comment:
Is this file needed? It doesn't look like it's used for anything.

Reply:
I don't think so, I added all of the files in the mrpPD folder to this PR

Reply:
I just removed the file